fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
+
+## Ordering the API Reference
+
+### Flattening sections
```

Comment:
You use the term "sections" 11 times on the page but I am unclear on what the definition is. Do you want to define what a "section" is up top?

Reply:
I tried definition the word "section" in the note [here](https://fern-preview-ef1e4cbd-7f22-4c16-9327-f1705dc2ce66.docs.buildwithfern.com/learn/docs/api-references/customize-api-reference-layout#ordering-top-level-sections), is there a better place to put this?